### PR TITLE
Update control buttons

### PR DESCRIPTION
**Pull request by maddiedierker**
This updates the control buttons to match Figma and moves the 'continue' button to a temporary spot (it and the 'settings' button will be moved/re-styled next week):

https://user-images.githubusercontent.com/9812299/120857449-2388b600-c536-11eb-8994-0614445cac74.mov

There's also some clean-up to:
- move `styles` definition below component definition
- move `<InputPrompt/>` into a separate file
- refactor the `isHorizontal` prop in `<JavalabButton/>` to actually make the icon/text horizontally stacked

## Links

- [CSA-361](https://codedotorg.atlassian.net/browse/CSA-361)
- [Figma](https://www.figma.com/file/fZo4fcZmDUioOCaz04Gza4/Javalab-Design-Spec?node-id=189%3A1435)

### CODE REVIEW

File: apps/style/javalab/style.scss [hunk 1]
```diff
@@ -8,15 +8,20 @@
   padding: 0;
 }
 
-.cm-content, .cm-gutter { 
+.cm-content,
```
Comment (maddiedierker) [inline apps/style/javalab/style.scss:11]: note: most of the changes in this file are formatting changes from prettier

File: apps/src/javalab/ControlButtons.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  // border is removed from common.scss.
+  return (
+    <div
+      style={{
+        ...styles.container,
+        backgroundColor: isDarkMode ? color.black : color.white
+      }}
+    >
+      <JavalabButton
+        text={isRunning ? 'Stop' : 'Run'}
```
Comment (jmkulwik) [inline apps/src/javalab/ControlButtons.jsx]: Should these be internationalized? (same question for other strings)

File: apps/src/javalab/ControlButtons.jsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-around'
+  },
+  button: {
+    all: {
+      fontSize: 16,
+      width: 140,
+      justifyContent: 'center',
```
Comment (jmkulwik) [inline apps/src/javalab/ControlButtons.jsx]: The justify-center feels pretty odd. Would left make more sense?

File: apps/src/javalab/ControlButtons.jsx (hunk 2 above)
Comment (maddiedierker) [reply apps/src/javalab/ControlButtons.jsx]: they weren't in i18n before, so i left them as-is, but i'll add them to the javalab/locale file

**Review by jmkulwik — APPROVED** [review]
LGTM!

**Pull request merged**